pyext/src/analysis_trajectories.py

```diff
@@ -536,28 +600,32 @@ def read_traj_info(self, out_dirs_sel):
             # Also add nuisances parameter
             sel_entries += [v for v in S_tot_scores.columns.values if 'Psi' in v and 'sum' not in v]
             
-            # Detect equilibration time
+            # Detect equilibration time if requested
             ts_eq = []
-            for r in sel_entries:
-                try:
-                    [t, g, N] = detectEquilibration(np.array(S_tot_scores[r].loc[self.th:]), nskip=self.nskip)
-                    ts_eq.append(t)
-                except:
-                    ts_eq.append(0)
-            print('Trajectory, ts_eqs: ',traj, ts_eq)
-            ts_max = np.max(ts_eq)+self.th
-            self.Sampling['N_equilibrated'] = self.Sampling['N_equilibrated'] + [n_frames - ts_max] 
+            if self.detect_equilibration:
+                for r in sel_entries:
+                    try:
+                        [t, g, N] = detectEquilibration(np.array(S_tot_scores[r].loc[self.th:]), nskip=self.nskip)
+                        ts_eq.append(t)
+                    except:
```

Comment:
I know this is in the original code too... but a bare `except` is rarely what you want. List the specific exceptions you want to catch instead so you don't skip everything on a `SyntaxError` for example.

Reply:
removed bare ```except``` s in ```analysis_trajectories.py``` as well ```equilibration.py```